service/docs/source/overview.rst

```diff
@@ -309,33 +312,31 @@ Reading Signals
 Understanding Aggregation
 """""""""""""""""""""""""
 
-The telemetry that is output from ``geopmread`` or the APIs will be
-automatically aggregated based on the requested domain and the aggregation
+The telemetry that is output from ``geopmread`` or the APIs will automatically
+be aggregated based on the requested domain and the aggregation
 type.
```

Comment:
We should probably add a second example with a different agg type so people understand what we mean by this. E.g., CPU_ENERGY uses a sum aggregator vs the time-weighted mean used by CPU_FREQEUNCY_STATUS.

The second example doesn't need as much detail as the first example's discussion. We just need to highlight that it is aggregated differently.